src/react/components/pages/editorPage/editorSideBar.tsx

```diff
@@ -77,8 +78,8 @@ export default class EditorSideBar extends React.Component<IEditorSideBarProps,
         }
     }
 
-    private getRowHeight = (width: number) => {
-        return width / (4 / 3) + 16;
+    private getColumnWidth = (height: number) => {
+        return (height - 16) * (4 / 3);
```

Comment:
Looks aspect ratio related (possibly?). Would be good to make these consts w/ good names, so they're less magic number-y :slightly_smiling_face:

Reply:
Fixed!